feat: better per svm docs (#536)
pyth-network/documentation: pages/express-relay/integrate-as-searcher/svm.mdx

SVM Express Relay searchers fulfill opportunities representing limit orders on the [Limo](https://solscan.io/account/LiMoM9rMhrdYrfzUCxQppvxCSG1FcrUK9G8uLq4A1GF) program.

Bids are represented via [SVM transactions](https://solana.com/docs/core/transactions) that include instructions for submitting the bid.

<Steps>

### Subscribe to SVM chain

Searchers can use [Typescript](https://github.com/pyth-network/per/tree/4be711525948cf24c0ebd4ebab007dc7f51b7069/sdk/js)
and [Python](https://github.com/pyth-network/per/tree/4be711525948cf24c0ebd4ebab007dc7f51b7069/sdk/python) SDKs to interact with auction server.
Searchers can also directly fetch available opportunities via HTTP or subscribe to them via WebSocket.

<Callout type="warning" emoji="⚠️">
  Polling the server via HTTP for new opportunities is not recommended because
  of the additional delay it may incur. This can lead to late bid submissions
  which will be rejected. Some submission information such as the most recent
  blockhash and priority fees are **only** broadcasted via websocket.
</Callout>

<Tabs items={['Typescript', 'Python', 'HTTP', 'WebSocket']}>

<Tabs.Tab>

```typescript

> import { Client, Opportunity } from "@pythnetwork/express-relay-js";

Reviewer: ```suggestion
import { ChainId, Client, Opportunity } from "@pythnetwork/express-relay-js";
```

> latestChainUpdate: Record<string, SvmChainUpdate> = {}

Reviewer: ```suggestion
const latestChainUpdate: Record<string, SvmChainUpdate> = {}
```

  "order": "UxMUbQAsjrfQUp5stVwMJ6Mucq7VWTvt4ICe69BJ8lVXqwM+0sysV8OqZTdM0W4p...",
  // Address of the order account
  "order_address": "DUcTi3rDyS5QEmZ4BNRBejtArmDCWaPYGfN44vBJXKL5",
  // Identifier of the program that the order exists in
  "program": "limo",
  "chain_id": "solana",
  // Opportunity format version
  "version": "v1",
  "opportunity_id": "271f2a7b-1ec5-420f-b5c0-e3e4317f3d7b",
  // Creation time of the opportunity (in microseconds since the Unix epoch)
  "creation_time": 1733503592579589,
  "slot": 305802439
}
```

The `order` field includes the [Limo](https://solscan.io/account/LiMoM9rMhrdYrfzUCxQppvxCSG1FcrUK9G8uLq4A1GF) program
order data that can be decoded using the SDKs provided or the program anchor idl. It includes all the necessary information
to fill the limit order:

- Maker address
- Input token and amount (what the maker is selling)
- Output token and amount (what the maker is buying in exchange for the input token)

<Callout type="info">
  Limo limit orders can also be filled partially in a linear fashion. For
  example, if the order is to buy 10 SOL for \$2000, you can provide 5 SOL and
  get back \$1000.
</Callout>

> The auction server also broadcast chain specific information that are necessary for building the transaction:

Reviewer: ```suggestion
The auction server also broadcasts chain specific information that is necessary for building the transaction:
```

from express_relay.models.svm import BidSvm
from express_relay.svm.limo_client import OrderStateAndAddress

> DEADLINE = 2 * 10**10

Reviewer: why this change?

{
  // serialized transaction object, in base-64 encoding
  "transaction": "SGVsbG8sIFdvcmxkIQ==",
  "chain_id": "solana"
}
```

The transaction submitted to the auction server as the bid should meet the following criteria:

- It should contain an Express Relay `SubmitBid` instruction that specifies the amount you are bidding, the permission details, and the deadline.
  This instruction can be created via the SDKs.
- The deadline specified in the `SubmitBid` instruction should be at least 5 seconds in the future.
- It should contain an instruction to set the [transaction priority fee](https://solana.com/developers/guides/advanced/how-to-use-priority-fees#what-are-priority-fees). The priority fee should be at least as large the amount
  advertised via websocket.

> - It should contain valid signatures for all signers except the relayer

Reviewer: ```suggestion
- It should contain valid signatures for all signers except the relayer.
```

> - It should pass simulation

Reviewer: ```suggestion
- It should pass simulation.
```
